.devcontainer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:20.04
 
 ARG TERRAFORM_VERSION=0.14.8
+ARG TARGETARCH
```

Comment:
Can you also include default arch?

Reply:
Arch is provided by docker depending on which architecture you are building for. 
See https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope